src/main/jbake/content/docs/dev/development.adoc

```diff
@@ -33,7 +33,8 @@ components to build the hawkular solution, this is the project that users would
 |Hawkular-Bus|https://github.com/hawkular/hawkular-bus|https://issues.jboss.org/browse/HWKBUS|Communication back bone
 |Hawkular-Inventory|https://github.com/hawkular/hawkular-inventory|https://issues.jboss.org/browse/HWKINVENT|In charge of modeling the various components to monitor and their relationships.
 |Hawkular-Build-Tools|https://github.com/hawkular/hawkular-build-tools|https://issues.jboss.org/browse/HAWKULAR|Various tools to help for the build, contains checkstyle definitions for instance
-|Hawkular-Accounts|https://github.com/hawkular/hawkular-accounts|https://issues.jboss.org/browse/HAWKULAR|Authorization framework for Hawkular components.
+|Hawkular-Accounts|https://github.com/hawkular/hawkular-accounts|https://issues.jboss
+.org/browse/HAWKULAR|Authorization framework for Hawkular components. link:accounts.html[More...]
```

Comment:
Is this the appropriate location for the link? Or would you suggest somewhere else?